code/src/App.js

```diff
@@ -1,9 +1,73 @@
-import React from 'react';
+/* eslint-disable max-len */
+import React, { useState, useEffect } from 'react';
+import { ThoughtInput } from 'components/ThoughtInput';
+import { EachThought } from 'components/EachThought';
 
 export const App = () => {
+  const [eachThought, setEachThought] = useState('');
+  // because a new thought has no value and then changes state to having a value
+
+  const [thoughtInput, setThoughtInput] = useState([]);
+  // because the Thought Input changes state when a new thought is added
+
+  const fetchThoughts = () => {
+    fetch('https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts')
+      .then((res) => res.json()) // converts the 'response' object to a JSON object
+      .then((data) => setThoughtInput(data)) // updates the state with the data from the
+    // response using the 'setThoughtInput' function
+      .catch((error) => console.error(error)) // catches errors
+      .finally(() => { console.log('fetch was successful') }) // this is where setLoading(false)); would go
+  }
+
+  useEffect(() => {
+    fetchThoughts();
```

Comment:
Is there any specific reason why you did a separate function instead of putting the fetch request directly in useEffect?